source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -79,6 +79,20 @@ Engine also covers the validation:
 - Physical network must be on the same Data Center as the external network.
 - The custom values (physical network and VLAN ID) must not be specified a when provider physical network is set.
 
+### Import of a provider physical network
+While importing a network with physical network access from an external network
+provider into Engine's database, Engine attempts to map this physical network has to be mapped to the corresponding logical network in Engine.
+The [external network provider][1] describes the physical network with the
+three attributes `provider:physical_network`, `provider:network_type` and
+`provider:segmentation_id`.  
+The corresponding logical network is detected in the context of a data center
+by checking if there is a logical network in the data center with the
+[VDSM name][2] equal to the `provider:physical_network` attribute, which is from
+the same network type equal to the `provider:network_type` attribute and has the same
+VLAN ID like the `provider:segmentation_id` attribute.
```

Comment:
Maybe:

The corresponding logical network is identified by matching the three attributes:
  `provider:physical_network` - [VDSM name][2]
  `provider:network_type` - type of network
  `provider:network_type` - VLAN ID

Reply:
Done